tests/health_misinfo_shared/test_claim_formatting.py

```diff
@@ -0,0 +1,148 @@
+                    "understandability": "loud and clear",
+                    "type_of_claim": "a bad one",
+                    "type_of_medical_claim": "a serious one",
+                    "support": "none at all",
+                    "harm": "too much",
+                },
+            },
+            False,
+            id="missing original_text",
+        ),
+        param(
+            {
+                "claim": "apples help you levitate",
+                "original_text": "eating an apple regularly will allow you to float away",
+                "labels": {
+                    "understandability": "loud and clear",
+                    "type_of_medical_claim": "a serious one",
+                    "support": "none at all",
+                    "harm": "too much",
+                },
+            },
+            False,
+            id="missing a label",
+        ),
+    ],
+)
+def test_assert_output_json_format(output, expected_bool):
+    assert assert_output_json_format(output) == expected_bool
+
+
+@mark.parametrize(
+    "output, expected_amended",
+    [
+        param(
+            {
+                "claim": "apples help you levitate",
+                "labels": {
+                    "understandability": "loud and clear",
+                    "type_of_claim": "a bad one",
+                    "type_of_medical_claim": "a serious one",
+                    "support": "none at all",
+                    "harm": "too much",
+                },
+            },
+            {
+                "claim": "apples help you levitate",
+                "original_text": "not found",
+                "labels": {
+                    "understandability": "loud and clear",
+                    "type_of_claim": "a bad one",
+                    "type_of_medical_claim": "a serious one",
+                    "support": "none at all",
+                    "harm": "too much",
+                },
+            },
+            id="original_text missing",
+        ),
+        param(
+            {
+                "claim": "apples help you levitate",
+                "original_text": "eating an apple regularly will allow you to float away",
+                "labels": {
+                    "understandability": "loud and clear",
+                    "type_of_claim": "a bad one",
+                    "support": "none at all",
+                    "harm": "too much",
+                },
+            },
+            {
+                "claim": "apples help you levitate",
+                "original_text": "eating an apple regularly will allow you to float away",
+                "labels": {
+                    "understandability": "loud and clear",
+                    "type_of_claim": "a bad one",
+                    "type_of_medical_claim": "",
+                    "support": "none at all",
+                    "harm": "too much",
+                },
+            },
+            id="one label missing",
+        ),
+        param(
```

Comment:
If the entire `labels` key is missing, is it replaced? Can we add a test item for that?

Reply:
This is the next test below. The output should be a label dict with just empty strings for each feature